frontend/packages/dev-console/integration-tests/features/build-config/shipwright-build/shipwright-build-detail-page.feature

```diff
@@ -0,0 +1,79 @@
+@odc-4981
+Feature: Shipwright build details page
+              As a user, I want check my Shipwright Build and see all related BuildRuns in a second tab.
+
+        Background:
+            Given user has installed OpenShift Pipelines Operator
+              And user is at developer perspective
+              And user has created or selected namespace "aut-shipwright-build-details"
+              And user has installed Shipwright Operator
+              And user is at Add page
+
+
+        @smoke
+        Scenario: Shipwright build page in dev perspective: SWB-01-TC01
+            Given user has created shipwright builds
+             When user navigates to build tab
+              And user clicks on "Shipwright Builds" tab
+             Then user will see Shipwright Builds
+              And user will see "Succeeded", "Failed" and "Unknown" in Filter list
+
+
+        @smoke
+        Scenario: Shipwright build page in admin perspective: SWB-01-TC02
+            Given user has created shipwright builds
+             When user switches to Administrative perspective
+              And user clicks on Builds tab in Administrative perspective
+             Then user will see "Shipwright Builds" tab
+             Then user will see "Shipwright BuildRuns" tab
+
+
+        @regression
+        Scenario: Shipwright build details page: SWB-01-TC03
+            Given user is on build tab in Developer perspective
```

Comment:
Just one sentence here? Would expect

Given user is in Developer perspective
And user is on the Builds navigation item

?

Reply:
This is to separate from the build tab in the Administrator perspective and moreover `user is in Developer perspective` is part of the background statement